ardrive_io/example/lib/main.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:ardrive_io/ardrive_io.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: ArDriveIOExample(),
+    );
+  }
+}
+
+class ArDriveIOExample extends StatefulWidget {
+  const ArDriveIOExample({Key? key}) : super(key: key);
+
+  @override
+  State<ArDriveIOExample> createState() => _ArDriveIOExampleState();
+}
+
+class _ArDriveIOExampleState extends State<ArDriveIOExample> {
+  String? fileDescription;
+  IOFile? currentFile;
+  IOFolder? currentFolder;
+
+  Future<void> pickFile() async {
+    final file = await ArDriveIO().pickFile();
+
+    setState(() {
+      currentFile = file;
+      currentFolder = null;
+    });
+  }
+
+  Future<void> pickFolder() async {
+    final folder = await ArDriveIO().pickFolder();
```

Comment:
whitespace nit, so it will be consistent with `pickFile`:

```suggestion
    final folder = await ArDriveIO().pickFolder();
    
```